### PR TITLE
Create part 2 - sport game.py

### PR DESCRIPTION
**Pull request by eladshlomy**

### CODE REVIEW

File: ex2/part 2 - sport game.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+def del_space1(text: str):
```
Comment (A4Vision) [inline ex2/part 2 - sport game.py:5]: This method is complicated and very hard to read.
Let's simplify it ! Hooray !

Follow the following steps * in order*:
1. Write in Hebrew what it does.
2. Rename the function to a more verbose name.
3. Write one line documentation in English.
4. Write tests for the function. The tests should cover all interesting cases and edge cases.
5. Then and only then (after we know what it should do AND we have tests in place) -
rewrite the function.
Use the following methods of string -
`replace`
`strip`
I reckon that one can implement the required functionality with only two simple lines.

File: ex2/part 2 - sport game.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    while len(guest_team_event_lst) > len(home_team_event_lst):
+        home_team_event_lst.append("")
+
+    while len(guest_team_event_lst) < len(home_team_event_lst):
+        guest_team_event_lst.append(" " * (len(home_team_event_lst[0]) + 2))
+
+    home_event_iter = iter(home_team_event_lst)
+    guest_event_iter = iter(guest_team_event_lst)
+    for i in range(len(home_team_event_lst)):
+        print(next(home_event_iter), next(guest_event_iter))
```
Comment (A4Vision) [inline ex2/part 2 - sport game.py:64]: Re write lines 61-64 using `zip`

File: ex2/part 2 - sport game.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            final_txt = final_txt[0:-1]
+
+        while final_txt[0].isspace():
+            final_txt = final_txt[1:]
+    except IndexError:
+        pass
+    return final_txt
+
+
+def main():
```
Comment (A4Vision) [inline ex2/part 2 - sport game.py:28]: What does this script do ?
Please include a few example outputs.

**Review by A4Vision — COMMENTED** [review]
This code gives us great examples for code simplifications (aka refactoring) opportunities !

After handling the comments above, I can give you more feedback if you like .

Why aren't you following the instructions that I give you ? Is it unclear ? Too hard ? Is it boring ? Are you intrigued with something else ?

Please explain - this submission contains non trivial logic and is missing tests. Therefore it is not good.
